controllers/api/userRoutes.js

```diff
@@ -5,12 +5,18 @@ router.post('/', async (req, res) => {
   try {
      const Userdata = await User.create(req.body)
 
+     req.session.save(() => {
+        req.session.user_id = Userdata.id;
+        req.session.logged_in = true;
```

Comment:
The group should talk about this - we don't really need the user's Id and logged_in. If we have the user's Id in the session then we already know they are logged in.